### DISCUSSION TITLE
Lazer tries to update despite being on latest version

### DISCUSSION THREAD

**Discussion by Rekunan** [ppy/osu#23151] [Q&A]
![image](https://i.imgur.com/h4Fau4w.jpeg)
[database.log](https://github.com/ppy/osu/files/11173369/database.log)
[input.log](https://github.com/ppy/osu/files/11173371/input.log)
[network.log](https://github.com/ppy/osu/files/11173372/network.log)
[performance.log](https://github.com/ppy/osu/files/11173373/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11173374/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11173375/updater.log)

**Comment by peppy** [comment]
```
2023-04-06 18:07:17 [error]: update failed!
2023-04-06 18:07:17 [error]: System.IO.IOException: The process cannot access the file 'C:\Users\Mangs\AppData\Local\osulazer\app-2023.403.1\AutoMapper.dll' because it is being used by another process.
```

The update is failing, likely because wither windows defender or a virus scanner on your pc is keeping the files open that are being attempted to update.
